packages/corvid-fake-local-mode-editor/src/browser/browser.html

```diff
@@ -3,5 +3,8 @@
 </head>
 <body>
   <script type='text/javascript' src='/editor.js'></script>
+  <script type='text/javascript'>
+    loadEditor() 
```

Comment:
the idea for doing this change was to pass the options from here instead of fetching window.location inside the browser.js file.
if you're not passing anything to loadEditor, you don't need this change.
decide which way is better

Reply:
done